cypress/integration/Slider.spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="Cypress" />
+
+/* eslint-disable no-undef */
+describe('The Slider component', () => {
+
+  beforeEach(() => {
+    // Setting viewport dimensions to avoid side effects
+    cy.viewport(1000, 660);
+  });
+
+  describe('when enabled', () => {
+    beforeEach(() => {
+      cy.visit('/iframe.html?selectedKind=Form%7CSlider&selectedStory=basic%20slider');
+    });
+
+    it('should update the sliders value when clicking on the track', () => {
+      cy.wait(200); // Gives slider some time for calculations
+
+      cy.get('[data-id="slider-wrapper"]').trigger('mousedown', { button: 0, pageX: 200 });
+
+      cy.wait(200); // Gives slider some time for calculations
```

Comment:
Ive given up

Reply:
I don't mind doing this - this is a relatively small library that won't have many tests - if it gets flaky, it can be removed. Maybe add a comment to that effect? Something like `// Remove if introducing flakiness` or something?